content/en/docs/installation/install-on-k3s.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Installing Clusternet on k3s"
+description: "How is the installation different on k3s"
+date: 2023-07-10
+draft: false
+weight: 4
+collapsible: false
+---
+
+Installing Clusternet in a [K3S](https://k3s.io/) cluster requires some additional configurations.
+
+#### Set anonymous-auth to false
+
+- If [installed via Helm](/docs/installation/install-with-helm), you need to set `anonymousAuthSupported` to `false` in the value files for both clusternet-hub and clusternet-controller-manager Charts.
```

Comment:
This flag is only available from clusternet-controller-manager starting from v0.15.0